erdpy/cli_contracts.py

```diff
@@ -121,6 +122,16 @@ def setup_parser(args: List[str], subparsers: Any) -> Any:
     _add_arguments_arg(sub)
     sub.set_defaults(func=query)
 
+    sub = cli_shared.add_command_subparser(subparsers, "contract", "reproducible-build",
+                                            "Build a Smart Contract and get the same output as a previously built Smart Contract")
+    _add_project_arg(sub)
+    _add_recursive_arg(sub)
```

Comment:
Here, I think we do not need the `_add_recursive_arg`. Remove if redundant.

Reply:
Deleted it!